### PR TITLE
Heptagon version 1.03.01

### PR DESCRIPTION
**Pull request by gdelaval**

### CODE REVIEW

File: packages/heptagon/heptagon.1.03.01/opam [hunk 1]
```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "gwenael.delaval@inria.fr"
+authors: "Gwenaël Delaval"
+homepage: "http://bzr.inria.fr"
+dev-repo: "git+ssh://scm.gforge.inria.fr//gitroot/heptagon/heptagon.git"
```
Comment (dsheets) [inline packages/heptagon/heptagon.1.03.01/opam]: The second `//` looks suspicious (but is probably harmless). I can't seem to clone the repo, with any combination of URL modifications. If the repo is not intended to be publicly accessible, please remove this field from the `opam` file.

Comment (gdelaval) [inline packages/heptagon/heptagon.1.03.01/opam]: The repository is indeed not yet intended to be publicly accessible. I remove this line.

Thanks.

**Comment by dsheets** [comment]
Thanks! Would it be possible to include the `heptagon.install` file in the upstream for the next release?

File: packages/heptagon/heptagon.1.03.00/opam [hunk 2]
```diff
@@ -9,10 +9,13 @@ build: [
...
+  "ocamlfind"  {build}
+  "menhir"     {build & >= "20141215"}
+  "ocamlgraph" {build}
+  "camlp4"     {build}
   "ocamlbuild" {build}
 ]
-depopts: ["lablgtk"]
+depopts: [
+  "lablgtk"
+  "reatk"
```
Comment (avsm) [inline packages/heptagon/heptagon.1.03.00/opam]: This `reatk` dependency is not present in the stable OPAM repository. Do you mind if I remove it from the central repository?

**Pull request merged**